Resources/Audio/Backmen/Items/handling/attributions.yml

```
@@ -0,0 +1,71 @@
+    - "helmet_pickup.ogg"
+    - "knife_drop.ogg"
+    - "knife_pickup.ogg"
+    - "matchbox_pickup.ogg"
+    - "multitool_pickup.ogg"
+    - "paper_drop.ogg"
+    - "paper_pickup.ogg"
+    - "phone_drop.ogg"
+    - "phone_pickup.ogg"
+    - "pillbottle_drop.ogg"
+    - "pillbottle_pickup.ogg"
+    - "ring_drop.ogg, 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)